tests/common/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ set(TARGET aoscommoncpp_test)
 # Sources
 # ######################################################################################################################
 
+add_definitions(-DAOS_CONFIG_MONITORING_SEND_PERIOD_SEC=2
```

Comment:
Add definition to separate section: see sm CMakeLists.txt

Reply:
Done